addons/website_sale/views/templates.xml

```diff
@@ -1366,25 +1386,6 @@
                                 This product has no valid combination.
                             </p>
                             <t t-call="website_sale.product_accordion"/>
-                            <div
-                                id="contact_us_wrapper"
-                                t-attf-class="{{'d-flex' if combination_info['prevent_zero_price_sale'] else 'd-none'}} oe_structure oe_structure_solo #{_div_classes}"
```

Comment:
Before this Contact us page was displayed even when ``_is_add_to_cart_possible` is false now it'll only display when _is_add_to_cart_possible is true. Is it expected?